### ISSUE TITLE
Add tables support

### ISSUE THREAD

**Issue by mdtrooper** [vinayak-mehta/present#108]
**Describe the bug**

It could be good to add tables support.

**Steps to reproduce the bug**

Any presentation with the table:

````

# A table

| aaaaa | e |  |
|:-:|-:|-|
| ee | bbbbb |  |
| ff | d | cccc |
```` 
**Expected behavior**

# A table

| aaaaa | e |  |
|:-:|-:|-|
| ee | bbbbb |  |
| ff | d | cccc |

**Comment by mdtrooper** [comment]
I think that it is a bug, it is a feature...but I can change it.

**Comment by rbharadwaj9** [comment]
I would agree that this is more of a feature than a bug. @mdtrooper are you working on this issue? I can take a stab at it if you want!

**Comment by mdtrooper** [comment]
> I would agree that this is more of a feature than a bug. @mdtrooper are you working on this issue? I can take a stab at it if you want!

No, sorry @rbharadwaj9 I am not working in this feature.
